core/src/gl/texture.cpp

```diff
@@ -155,18 +155,16 @@ void Texture::update(GLuint _textureUnit) {
 
         m_shouldResize = false;
 
-        m_subData.clear();
+    }
 
```

Comment:
In this case we can just drop them since the old region will be set dirty and copied to subbuffers on the next call to FontContext::renderUpdate. Though I think with a slight change to texture and fontstash we only need one buffer and only copy instead of three + x (since also the subdata buffers can become quite large as one rasterize call can cause an update texture of the whole texture when the new glyphs are inserted at the top and bottom)